### DISCUSSION TITLE
QSplitter slots issue

### DISCUSSION THREAD

**Discussion by Dimonka2** [quasarframework/quasar#12142] [General - Components / Directives / etc]
I am using QSplitter with a QScrollArea in each slot and getting a weird overflow issue of slot elements (div.q-splitter__panel) - they do show native scrollbars and hide them on resize. Sometimes I am getting even an infinite loop of show/hide scrollbars.

I tried to make something like following, but QSplitter seems to ignore slot style:
`<template v-slot:after style="overflow: hidden;">`
 
Is there a workaround or any update planned? 
Thanks!

**Comment by pdanpdan** [comment]
Please add a codepen to show the issue

**Comment by Dimonka2** [comment]
Hmm, I was trying to mimic similar issue in codepen, but was not able to reproduce!
My issue is actually happens in Electron application.

**Reply by pdanpdan** [reply]
But a reproduction codepen would help

**Comment by Dimonka2** [comment]
I tried to reproduce it in codepen, but without success. After some research I found a solution. Just adding following style correction fixes this issue:

```css
  .q-splitter__panel {
    overflow: hidden;
  }
```
